x-pack/scripts/functional_tests.js

```diff
@@ -21,6 +21,7 @@ const onlyNotInCoverageTests = [
   require.resolve('../test/alerting_api_integration/spaces_only/config.ts'),
   require.resolve('../test/alerting_api_integration/security_and_spaces/config.ts'),
   require.resolve('../test/detection_engine_api_integration/security_and_spaces/config.ts'),
+  require.resolve('../test/detection_engine_api_integration/basic/config.ts'),
```

Comment:
NOTE: This adds it as the unit test to the overall unit tests. Check within the Jenkins output that you do indeed see messages related to the tests outputting success messages